themes/beaver/layouts/partials/img/homepage.html

```diff
@@ -0,0 +1,20 @@
+{{ $image := .image }}
+{{ $title := .title }}
+{{ $widths := slice 300 730 }}
+
+{{ $originalWidth := $image.Width }}
+{{ $imageWebP := $image.Resize (printf "%dx webp q75" $originalWidth) }}
+{{ $srcset := "" }}
+{{ range $widths }}
+{{ if lt . $originalWidth }}
+{{ $resized := $image.Resize (printf "%dx webp q75" .) }}
+{{ $srcset = printf "%s%s %dw, " $srcset $resized.RelPermalink . }}
+{{ end }}
+{{ end }}
+{{ $srcset = printf "%s%s %dw" $srcset $imageWebP.RelPermalink $originalWidth }}
+
+<img fetchpriority="high" class="fl-photo-img size-full"
+     src="{{ $imageWebP.RelPermalink }}" alt="{{ $title }}"
+     height="{{ $image.Height }}" width="{{ $image.Width }}" title="{{ $title }}"
+     srcset="{{ $srcset }}"
+     sizes="(max-width: 730px) 100vw, 730px">
```

Comment:
Can we pass this as a parameter?

Reply:
There is also sizes to render.
Imo I can remove duplication in next PR
I heard you liked small PRs